vpd_tool_impl.cpp

```diff
@@ -914,6 +913,90 @@ void VpdTool::parseSVPDOptions(const nlohmann::json& json)
                            "record-keyword pair. Exit successfully.\n";
                 }
 
+                exit(0);
```

Comment:
why is this case moved above ? can you revert this so that the diff is better

Reply:
Right, it is showing too much of difference. and not getting what is the actual change here.
Please revert this.

Reply:
I'll revert it.